# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/wlroots/hotfix-makedev.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- wlroots-0.16.0/render/vulkan/vulkan.c.vanilla	2022-11-15 15:38:01.431000000 +0100
+++ wlroots-0.16.0/render/vulkan/vulkan.c	2022-11-15 15:38:24.586000000 +0100
@@ -4,6 +4,7 @@
 #include <stdint.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <unistd.h>
 #include <xf86drm.h>
 #include <vulkan/vulkan.h>
